UnitTests/GitUITests/Script/ScriptRunnerTests.cs

```diff
@@ -186,9 +186,10 @@ public void RunScript_with_arguments_with_s_option_with_RevisionGrid_with_select
 
             RunFormTest(formBrowse =>
             {
-                // wait until the revisions are loaded
-                while (formBrowse.RevisionGridControl.LatestSelectedRevision == null)
+                // wait up to 5 seconds until the revisions will have been loaded
```

Comment:
📝 Waiting for a time limit should be abstracted, using either a `TimeSpan` or a `CancellationTokenSource` to represent an "unexpected timeout".

Reply:
done